packages/sdk/src/oracle/types/interfaces.ts

```diff
@@ -1,17 +1,54 @@
 import { BigNumberish, BigNumber, Signer, TransactionResponse, TransactionReceipt, Provider } from "../types/ethers";
-import { RequestState, RequestKey, Request as RequestFromEvent } from "../../clients/optimisticOracleV2";
+import { RequestState, RequestKey } from "../../clients/optimisticOracle";
 import { Client } from "../client";
 import { OracleType } from "../types/state";
 
+// This object diverges from the v1 and v2 request shapes. This is to normalize
+// the data into a common shape. This has pitfalls and will be problematic if oracle implementations
+// diverge too much, but so far it seems ok
+export type Request = RequestKey & {
```

Comment:
this is the new request object interface.  its not really changed that much, but no longer has a requestSettings object, all fields are flattened on to the request instead

Reply:
That makes sense! that way you always have a common single-layer object for all OO types.